types/chart.js/index.d.ts

```diff
@@ -601,6 +602,7 @@ declare namespace Chart {
     }) => T;
 
     interface ChartDataSets {
+        hoverRadius?: number;
```

Comment:
Alphabetical order please.

Also:
> Provide a URL to documentation or source code which provides context for the suggested changes:

You haven't filled in the template properly.

There is already a `hoverRadius` on `ChartPointOptions`. Are they not the same?

Reply:
Not its not the same